mix.exs

```diff
@@ -13,7 +13,11 @@ defmodule ElixirBase.Mixfile do
       escript: [main_module: ElixirBase.CLI],
       docs: [extras: ["README.md"], output: "./doc/app"],
       deps: deps(),
-      aliases: aliases()
+      aliases: aliases(),
+      dialyzer: [
+        plt_add_apps: [:mix],
+        flags: [:unmatched_returns, :error_handling, :race_conditions, :no_opaque, :underspecs]
```

Comment:
As of 0.4, there are no longer any flags used by default. To get the old behavior flags should be specified manually